src/runner/mod.rs

```diff
@@ -253,8 +257,10 @@ where
 
     fn visit(&self, example: &Example<T>, environment: &mut Self::Environment) -> Self::Output {
         self.broadcast(|handler| handler.enter_example(self, &example.header));
+        let start = PreciseTime::now();
```

Comment:
start_time and end_time, let's be precise

Reply:
Fixed.